dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -1024,6 +1030,39 @@ LanguageClientInterface convertCLIStringToEnum(String descriptor) {
         throw new UnsupportedOperationException("language not supported yet");
     }
 
+    /**
+     * Loads docker images from file system if there are any
+     */
+    public void loadDockerImages() {
```

Comment:
protected might be sufficient unless this is used somewhere either than subclasses

Reply:
The other clients have an abstractEntryClient instead of extending it so I get a `loadDockerImages() has protected access in io.dockstore.client.cli.nested.AbstractEntryClient` error during compiling